hw/top_earlgrey/BUILD

```
@@ -249,39 +249,61 @@ fpga_cw305(
 ###########################################################################
 fpga_cw340(
     name = "fpga_cw340",
+    args = [
+        "--rcfile=",
+        "--logging=info",
+        "--interface={interface}",
+    ] + select({
+        "@//ci:lowrisc_fpga_cw340": ["--uarts=/dev/ttyACM_CW340_1,/dev/ttyACM_CW340_0"],
+        "//conditions:default": [],
+    }),
     design = "earlgrey",
     exec_env = "fpga_cw340",
     libs = [
         "//sw/device/lib/arch:boot_stage_rom_ext",
         "//sw/device/lib/arch:fpga_cw340",
     ],
     linker_script = "//sw/device/lib/testing/test_framework:ottf_ld_silicon_creator_slot_a",
+    param = {
+        "interface": "hyper340",
+        "exit_success": DEFAULT_TEST_SUCCESS_MSG,
+        "exit_failure": DEFAULT_TEST_FAILURE_MSG,
+    },
     rom_scramble_config = "//hw/top_earlgrey/data:autogen/top_earlgrey.gen.hjson",
     test_cmd = "testing-not-supported",
 )
 
 fpga_cw340(
     name = "fpga_cw340_test_rom",
     testonly = True,
-    args = [
-        "--rcfile=",
-        "--logging=info",
-        "--interface={interface}",
-    ] + select({
-        "@//ci:lowrisc_fpga_cw340": ["--uarts=/dev/ttyACM_CW340_1,/dev/ttyACM_CW340_0"],
-        "//conditions:default": [],
-    }),
     base = ":fpga_cw340",
     base_bitstream = "//hw/bitstream/cw340:bitstream",
     exec_env = "fpga_cw340_test_rom",
     otp_mmi = "//hw/bitstream/cw340:otp_mmi",
```

It would be wise not to assume the bitstream's built-in OTP image is correct.

```suggestion
    otp_mmi = "//hw/bitstream/cw340:otp_mmi",
    otp = "//hw/ip/otp_ctrl/data:img_rma",
```